biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -1,18 +1,38 @@
-import { Battery, Energy, Validatable, validate } from "@biogrid/grid-simulator";
+import {
+  Battery,
+  Energy,
+  Validatable,
+  validate,
+  Distance,
+  ItemPosition,
+} from '@biogrid/grid-simulator';
 
 export class BioBattery implements Battery {
   private currentBatteryPower: Energy;
   private readonly maxCapacity: Energy;
+  private position: ItemPosition;
 
-  constructor(currentBatteryPower: Energy = 0, maxCapacity: Energy = 10) {
+  constructor(
+    x: Distance,
+    y: Distance,
```

Comment:
Can the position be negative relative to the grid, or is this the absolute distance (positive) for the item? I am asking because if it is negative, the graph can be influenced.

Reply:
Ok sure, I was thinking just positive. I added the comment to the code